src/index.html

```diff
@@ -10,8 +10,165 @@
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="birthDate">
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="agreement">
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            name="love_cats"
+            value="Yes"
+          >
+            Yes
+
+          <input
+            type="radio"
+            name="love_cats"
+            value="No"
+          >
+            No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="favoriteColor">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="timeToBed">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="brandOfCar"
+            required
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rating"
+            value="0"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
checklist p.8  - Check the textarea behaviour - when you click on the textarea, the coursor should appear on the beginning of the input.